packages/types/src/sdk/events/auth.ts

```diff
@@ -7,10 +7,12 @@ export type SSOType = ConfigType.OIDC | ConfigType.GOOGLE
 export interface LoginEvent extends BaseEvent {
   userId: string
   source: LoginSource
+  email: string
```

Comment:
Similar as above, we already enrich the user on read, do we need email here at all?

Reply:
It is used for generating the proper friendly names, as per the requirements - if we did not do this then the email address and app names would not be searchable.